src/test/unit/common/services/learnService.test.ts

```diff
@@ -0,0 +1,113 @@
+import { deepStrictEqual, strictEqual } from 'assert';
+import axios from 'axios';
+import sinon from 'sinon';
+import { OpenCommandIssueType } from '../../../../snyk/common/commands/types';
+import { LearnService } from '../../../../snyk/common/services/learnService';
+import type { completeFileSuggestionType } from '../../../../snyk/snykCode/interfaces';
+import { OssIssueCommandArg } from '../../../../snyk/snykOss/views/ossVulnerabilityTreeProvider';
+import { LoggerMock } from '../../mocks/logger.mock';
+
+const ossIssueCommandArgFixture = { identifiers: { CWE: ['CWE-1'] }, packageManager: 'npm' } as OssIssueCommandArg;
+const codeIssueCommandArgFixture = {
+  cwe: ['CWE-2'],
+  id: 'javascript%2Fdc_interfile_project%2FSqli',
+} as completeFileSuggestionType;
+const lessonFixture = {
+  title: 'lesson title',
+  lessonId: 'id',
+  ecosystem: 'javascript',
+  url: 'https://example.com',
+};
+
+teardown(() => {
+  sinon.restore();
+});
+
+suite('OSS functionality', () => {
```

Comment:
I'd call this suite "Learn Service" for both OSS and Code, otherwise it's ambiguous with OSS Service that we have for OSS scans.

Reply:
👍 I'll put a wrapper suite around all suites here